fhirstore/fhirstore.py

```diff
@@ -353,8 +370,7 @@ def search(self, resource_type, args):
                             f"{e.info['error']['index']} is not indexed in the database yet."
                         )
 
-            bundle = self.formatter.add_included_bundle(parsed_args, bundle, included_hits)
-
+            bundle.append_bundle(search_args.formatting_args, included_hits)
```

Comment:
bundle.append(). Also (little detail) it makes more sense to pass `included_hits` as the first argument, that way :
`bundle.append(hits, formatting args)` reads "append these hits to the bundle using this format"